File: SinaWeibo/ViewControllers/WBWebViewController.m
//
//  WBWebViewController.m
//  SinaWeibo
//
//  Created by cxjwin on 13-9-25.
//  Copyright (c) 2013年 cxjwin. All rights reserved.
//

#import "WBWebViewController.h"

@interface WBWebViewController ()

@end

@implementation WBWebViewController

- (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil
{
    self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil];
    if (self) {
        // Custom initialization
    }
    return self;
}

- (void)viewDidLoad
{
    [super viewDidLoad];
	// Do any additional setup after loading the view.
    
    self.webView.scalesPageToFit = YES;
}

- (void)viewWillAppear:(BOOL)animated
{
    [super viewWillAppear:animated];
    NSURLRequest *request = [NSURLRequest requestWithURL:[NSURL URLWithString:self.urlStr]];
    [self.webView loadRequest:request];
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: SinaWeibo/ViewControllers/WBWebViewController.swift
//
//  WBWebViewController.swift
//  SinaWeibo
//
//  Created by cxjwin on 6/13/14.
//  Copyright (c) 2014 cxjwin. All rights reserved.
//

import UIKit

class WBWebViewController: UIViewController {

	@IBOutlet var webView : UIWebView?
	
	var URLString: String?

    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
    }

	override func viewWillAppear(animated: Bool) {
		super.viewWillAppear(animated)
		
		// load request
		if let URLString = self.URLString {
			var URL = NSURL(string: URLString)
			var request = NSURLRequest(URL: URL)
			self.webView?.loadRequest(request)
		} else {
			println("url string is nil...")
		}
	}
	
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    

    /*
    // #pragma mark - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepareForSegue(segue: UIStoryboardSegue?, sender: AnyObject?) {
        // Get the new view controller using [segue destinationViewController].
        // Pass the selected object to the new view controller.
    }
    */

}
